### ISSUE TITLE
heaps of containers and images missing or superfluous in details panel

### ISSUE THREAD

**Issue by rade** [weaveworks/scope#1090]
most of the time i see this...
![screenshot from 2016-03-01 23 05 12](https://cloud.githubusercontent.com/assets/109109/13445132/66666bbc-e002-11e5-80ac-8f081188cee2.png)

occasionally it switches briefly to this...
![screenshot from 2016-03-01 23 05 55](https://cloud.githubusercontent.com/assets/109109/13445129/61ace48e-e002-11e5-8d1f-b7472366ae47.png)

I am running the probe on that host with

```
scope launch --no-app --weave.router.addr= --publish.interval=1s 192.168.48.1
```

and the app is run on another host with
```
scope launch --no-probe --app.window=3s
```

Could my tweaked interval/window settings be responsible for the odd behaviour?

**Comment by rade** [comment]
Note that *neither* view is correct! The actual containers running on that machine are:

```
$ docker ps -a
CONTAINER ID        IMAGE                         COMMAND                  CREATED             STATUS                      PORTS               NAMES
8f8dcc0ac5b7        iptables                      "iptables -D OUTPUT -"   About an hour ago   Exited (0) 40 minutes ago                       part_heal
96801435723e        iptables                      "iptables -A OUTPUT -"   About an hour ago   Exited (0) 20 minutes ago                       part_create
cd049728dd98        ubuntu                        "/w/w /bin/bash"         2 hours ago         Up 42 minutes                                   bar
543bc1f50a46        ubuntu                        "/w/w /bin/bash"         5 hours ago         Up 5 hours                                      foo
e13f22ecae20        weaveworks/plugin:latest      "/home/weave/plugin"     5 hours ago         Up 5 hours                                      weaveplugin
ac9c9cd7457a        weaveworks/weaveexec:latest   "/home/weave/weavepro"   5 hours ago         Up 5 hours                                      weaveproxy
b6694ee26428        weaveworks/weaveexec:latest   "/bin/false"             5 hours ago         Created                                         weavevolumes-latest
931c4e1b04ce        weaveworks/weave:latest       "/home/weave/weaver -"   5 hours ago         Up 5 hours                                      weave
366db82b2f67        weaveworks/scope:0.13.0       "/home/weave/entrypoi"   5 hours ago         Up 5 hours                                      weavescope
```

The second screenshot seems to show the containers across *all* machines (I do indeed have three scopes, three foos, two bars, etc in total)

**Comment by 2opremio** [comment]
Dupe of #1064 ?

**Comment by rade** [comment]
> Dupe of #1064 ?

The missing details bit perhaps, though in #1064 *all* containers etc disappear whereas here there are still some left.

The "showing details that don't belong here" bit not.

**Comment by tomwilkie** [comment]
Needs investigation, but I suspect this is due to @rade changing the periods, probes timing out and the tagger bug on shortcut reports.

**Comment by tomwilkie** [comment]
Testing with a multi host k8s cluster, all the right nodes appear in the right places.  I believe this was fixed mainly by #1113 (which would have meant containers are attributed to the wrong hosts), performance improvements and not messing with the timing,
